_extras/guide.md

```diff
@@ -1,6 +1,34 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+## Miscellaneous Tips
+
+* **Timing**: As written, there's way more than 3 hours of material in this lesson.
+Focusing on the earlier episodes (Introduction through the first half
+of Creating Container Images) will take just about three hours if you
+also include a brief general introduction and time to check people's
+installations.
+* **Install Issues**: Windows computers seem to consistently have issues with Docker.
+Consider having people check their install in advance at a separate time or come early.
+In online workshops, consider using breakout rooms to have smaller groups for
+participants to demonstrate their installations. Use a more complex command than
+`docker --version` to test the installation.
+* **Virtualization Illustration**: When going through the intro to containers,
+consider demonstrating what this might look like by having two shells (or shell tabs)
+open, one on your host computer and one into a container you started before the
+workshop. Then you can demonstrate in a simple way that from the same (host) computer,
```

Comment:
Maybe add in a suggested example command like `whoami` or `pwd`?

Reply:
Good idea! I tried to do a sub-list with ideas, am not sure the formatting will work.